roles/openshift_control_plane/handlers/main.yml

```diff
@@ -10,23 +10,6 @@
   until: result.rc == 0
   notify: verify API server
 
-- name: verify API server
-  # Using curl here since the uri module requires python-httplib2 and
-  # wait_for port doesn't provide health information.
-  command: >
-    curl --silent --tlsv1.2 --max-time 2
-    --cacert {{ openshift.common.config_base }}/master/ca-bundle.crt
-    {{ openshift.master.api_url }}/healthz/ready
-  args:
-    # Disables the following warning:
-    # Consider using get_url or uri module rather than running curl
-    warn: no
-  register: l_api_available_output
-  until: l_api_available_output.stdout == 'ok'
-  retries: 120
-  delay: 1
-  changed_when: false
-
```

Comment:
Was the idea to remove the verification check entirely? Or to change against whom the check was run (LB vs API)?